rpc_jobs/unit/re_unit_tests.yml

```diff
@@ -1,3 +1,41 @@
+- job:
+    name: 'RPC-Gating-Unit-Tests-gh'
```

Comment:
I assume you intend to remove this before merge?

Reply:
Yes, that's just for testing. It's not yet ready for review. I'd rather see the unit test job modified separately so as not to confuse things.